docs/docker.md

```diff
@@ -21,6 +21,20 @@ You should see the console output of the running node and a lot of information r
 
 [Verify if your node is connected to mainnet.](operation.md#verify-that-node-is-connected-to-the-mainnet)
 
+## Testnet
+
+To join the testnet a network_id with value `ae_uat` argument must be pass:
+
+
+```bash
+docker run -p 3013:3013 -p 3015:3015 aeternity/aeternity -network_id ae_uat
+```
+
+You should see the console output of the running node and a lot of information related to syncing with the network.
+
+[Verify if your node is connected to mainnet.](operation.md#verify-that-node-is-connected-to-the-testnet)
```

Comment:
Mainnet? Testnet? It seems there may be some incorrect testnet/mainnet refs in doc.

Reply:
Ah, I was testing it for consistency, will fix.